src/store/education/actions.ts

```diff
@@ -0,0 +1,21 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+import { IAddEducationPayload } from 'src/common/types';
+import { educationApi } from 'src/services';
+import { ActionTypes } from './common';
+
+const addEducation = createAsyncThunk(
+  ActionTypes.ADD_EDUCATION,
+  (payload: IAddEducationPayload) => {
+    return educationApi.addEducation(payload);
+  }
+);
+
+const loadEducationExperience = createAsyncThunk(
+  ActionTypes.LOAD_EDUCATION,
+  async () => {
+    return await educationApi.loadEducations();
```

Comment:
You can safely omit `await` here.
```suggestion
    return educationApi.loadEducations();
```